custom_components/opensprinkler/__init__.py

```diff
@@ -367,6 +367,14 @@ def state(self):
         return self._state
 
 
+class OpenSprinklerControllerEntity:
+    def stop(self):
```

Comment:
I'd probably name this a little more explicitly. `stop_stations` and then if we implement the other method `reset_stations`?

Reply:
I've just updated the lib to have both methods. I think we should have both services.

Reply:
I didn't realise there was that method. Is there a scenario where we would want to stop all stations but not the waiting? Maybe the reset should be enough?

Reply:
Yeah, good question. I've been surprised by the general sense that folks are wanting to build some pretty power scheduling/etc logic outside the unit itself (I may be misreading the sentiment). My comment is mostly driven by that feeling.

In that same vein I'd suggest we also add `enable`/`disable` services for the controller and possible even `reboot`.

Reply:
The controller already has an enable/disable switch, I don't see why it needs a service. The reboot service we can add if it is requested, but I don't see the need to add things that potentially might never get used.

Reply:
For this PR I think we should just keep the one service

Reply:
That's fair...haven't delved into the automations too deeply yet so I'm a bit blind on this stuff. Meaning, if switches can generally be toggled in automations and don't require services then yeah, agreed.